ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -150,6 +159,7 @@ def main(args: DatasetToolArgs):
         "token": token,
         "revision": args.upload_branch,
         "private": args.private,
+        "split": args.upload_split,
```

Comment:
Wait, you didn't fix the conflict here!
Now on line 170 you'll be sending the `split` argument twice, and with different values!